src/harness.js

```diff
@@ -794,6 +789,29 @@ const resolveMock = async (coreAdapter, state, mock, options = {}) => {
   return mock;
 };
 
+const prepareTasks = async (coreAdapter, state, options) => {
+  const user = await resolveMock(coreAdapter, state, options.user);
+  const subject = await resolveMock(coreAdapter, state, options.subject, { hydrate: false });
+  const tasks = await coreAdapter.fetchTasksFor(user, options.userRoles, stateEnsuringPresenceOfMocks(state, user, subject));
+  const allTaskDocs = await coreAdapter.fetchTaskDocs(); // Should come after tasks
+
+  tasks.forEach(task => task.emission.actions.forEach(action => {
+    action.forId = task.emission.forId; // required to hydrate contact in loadAction()
+  }));
+
+  const tasksEmissionLog = _.countBy(filterTaskDocs(allTaskDocs, subject._id, options), (taskDoc) => {
```

Comment:
why the call to `filterTaskDocs`?  if the console.warn feature is about alerting users when there is an unhealthy task - should it look at the unfiltered task documents?

Reply:
yeah i have it unfiltered on my PC just hadn't push to remote :)

Reply:
though i still filter by subject at least guess we can ditch that as well